.azure-pipelines/azure-pipelines-UpgradeVersion.yml

```diff
@@ -47,12 +62,16 @@ stages:
         echo "artifacts$artifacts"
         cp -r $(Pipeline.Workspace)/sonic-buildimage.${default_platform}/versions target/
         make freeze FREEZE_VERSION_OPTIONS=-r
-        find files/build/versions 
-        for artifact in $artifacts
+        find files/build/versions
+        ordered_artifacts=$(echo "$artifacts" | grep -v -E "arm64|armhf" && echo "$artifacts" | grep -E "arm64|armhf")
+        for artifact in $ordered_artifacts
         do
+          [[ "$artifact" == *arm64* || "$artifact" == *armhf* ]] && continue
           rm -rf target/versions
           cp -r $artifact/versions target/
-          make freeze FREEZE_VERSION_OPTIONS="-a -d"
+          OPTIONS="-a -d"
+          [[ "$artifact" == *arm64* || "$artifact" == *arm64* ]] && OPTIONS="-d"
```

Comment:
Remove line 69, and change line 73 to
 [[ "$artifact" == *arm64* || "$artifact" == *armhf* ]] && OPTIONS="-d"